data_tools/add.py

```diff
@@ -7,6 +7,8 @@
 import validate_message_structure
 import validate_user_list
 
+BATCH_SIZE = 500
```

Comment:
I don't think you're referencing this anywhere. Are you planning to use it later or can you delete it?

Reply:
Thanks